src/test/java/uk/gov/companieshouse/overseasentitiesapi/validation/RemoveValidatorTest.java

```diff
@@ -0,0 +1,71 @@
+package uk.gov.companieshouse.overseasentitiesapi.validation;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import uk.gov.companieshouse.overseasentitiesapi.mocks.RemoveMock;
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.RemoveDto;
+import uk.gov.companieshouse.service.rest.err.Err;
+import uk.gov.companieshouse.service.rest.err.Errors;
+import uk.gov.companieshouse.overseasentitiesapi.validation.utils.ValidationMessages;
+import static uk.gov.companieshouse.overseasentitiesapi.model.dto.OverseasEntitySubmissionDto.REMOVE_FIELD;
+
+
+@ExtendWith(MockitoExtension.class)
+class RemoveValidatorTest {
+
+    private static final String LOGGING_CONTEXT = "12345";
+
+    private RemoveValidator removeValidator;
+    private RemoveDto removeDto;
+
+    @BeforeEach
+    public void init() {
+        removeValidator = new RemoveValidator();
+        removeDto = RemoveMock.getRemoveDto();
+    }
+    @Test
+    void testNoValidationErrorReportedWhenIsNotProprietorOfLandIsNull() {
+        removeDto.setIsNotProprietorOfLand(null);
+        Errors errors = removeValidator.validate(removeDto, new Errors(), LOGGING_CONTEXT);
+        assertFalse(errors.hasErrors());
+    }
+
+    @Test
+    void testNoValidationErrorReportedWhenIsNotProprietorOfLandIsNotNull() {
```

Comment:
This test name might not be correct, ...IsNotNull , as it looks like  isNotProprietorOfLand is null as the init() does not set IsNotProprietorOfLand to be a value.

Reply:
perhaps this test is not needed? @mwestacott do you think this test is basically the same as the one above?

Reply:
I agree @markpit , I don't think the test is needed as when it's null the test above takes care of it and when it's not null it has to have a value of either true or false, which is covered by the two tests below. So, yes, I think remove it.

Reply:
test removed